OpenQA/Isotovideo/Dewebsockify.pm

```diff
@@ -0,0 +1,125 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+
+package OpenQA::Isotovideo::Dewebsockify;
+
+use Mojo::Base -strict, -signatures;
+use Mojo::IOLoop::Server;
+use Mojo::IOLoop::Stream;
+use Mojo::Log;
+use Mojo::UserAgent;
+
+sub main ($args) {
```

Comment:
As @Martchus already commented [here](https://github.com/os-autoinst/os-autoinst/pull/2598/files#r1910244275), separate function would be nice, but not in the context of this pr. @okurz in those case we open a new task in progress?

Reply:
yes, good idea. please do that

Reply:
https://progress.opensuse.org/issues/175335